nevergrad/optimization/optimizerlib.py

```diff
@@ -1021,6 +1021,7 @@ def __init__(
         num_workers: int = 1,
         base_optimizer: base.OptCls = CMA,
         scale: tp.Optional[float] = None,
+        translate: tp.Optional[float] = None,
```

Comment:
This would be more correct
```suggestion
        translation: float = 0.0,
```